gtsfm/runner/gtsfm_runner_base.py

```diff
@@ -12,9 +12,16 @@
 from gtsfm.loader.loader_base import LoaderBase
 from gtsfm.scene_optimizer import SceneOptimizer
 
+from gtsfm.retriever.joint_netvlad_sequential_retriever import JointNetVLADSequentialRetriever
+from gtsfm.retriever.netvlad_retriever import NetVLADRetriever
+from gtsfm.retriever.sequential_retriever import SequentialRetriever
+
 
 logger = logger_utils.get_logger()
 
+# For exhaustive matching, we limit the lookahead to 10,000 images.
```

Comment:
Out of curiosity, how was this limit decided? I would think the effect of additional matches to a particular image would become marginal before 10,000 images.

Reply:
Was just an arbitrary number that I didn't think we could reach / handle